org.civicrm.reqangular/src/common/services/session.js

```diff
@@ -1,12 +1,15 @@
 /* eslint-env amd */
-/* global Drupal */
+/* global CRM */
 
 (function (CRM, Drupal) {
   define([
     'common/modules/services'
   ], function (services) {
     'use strict';
 
+    // Default Drupal value in case it's not defined
+    if (typeof Drupal === 'undefined') Drupal = { settings: {} };
```

Comment:
if block should always be used with curly braces, but in this case we can simply do
`Drupal = Drupal || { settings: {} };`